### PR TITLE
Added one extra example in the authorized_key module.

### PR DESCRIPTION
**Pull request by marcosdiez**
##### ISSUE TYPE
- Docs Pull Request
##### COMPONENT NAME

authorized_key module
##### ANSIBLE VERSION

```
ansible 2.2.0 (detached HEAD b4e78caf25) last updated 2016/05/12 15:28:18 (GMT -200)
  lib/ansible/modules/core: (doc_authorized_keys 8522dec1c1) last updated 2016/05/12 15:26:20 (GMT -200)
  lib/ansible/modules/extras: (mongodb_query dd2daac64a) last updated 2016/03/20 00:52:20 (GMT -200)
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides

```
##### SUMMARY

Just adding one extra example on the authorized_key module
It took me a few minutes to find that case by myself and it will hopefully save many minutes from others.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @marcosdiez for this PR. This module is maintained by the Ansible core team, so it can take a while for patches to be reviewed. Thanks for your patience.

Core team: please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with 'needs_revision' or merge as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: system/authorized_key.py [hunk 1]
```diff
@@ -124,6 +124,14 @@
 - authorized_key: user=root key="{{ item }}" state=present exclusive=yes
   with_file:
     - public_keys/doe-jane
+
+# Copies the key from the user who is running ansible to the remote machine
+- set_fact:
+    my_ssh_key: "{{ lookup('env','HOME') }}/.ssh/id_rsa.pub"
+
+- authorized_key: user=ubuntu key="{{ lookup('file', my_ssh_key) }}"
+  sudo: yes
```
Comment (alxgu) [inline system/authorized_key.py:133]: Isn't sudo deprecated? Better use `become: yes`

Comment (mscherer) [inline system/authorized_key.py:133]: Indeed. Do you wish to make a PR for that ?

Comment (marcosdiez) [inline system/authorized_key.py:133]: I'll do it.

Comment (marcosdiez) [inline system/authorized_key.py:133]: done. https://github.com/ansible/ansible-modules-core/pull/3739

**Pull request merged**